lib/rubocop/cop/layout/multiline_method_parameter_line_breaks.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Layout
+      # Ensures that each argument in a multi-line method call
+      # starts on a separate line.
+      #
+      # NOTE: this cop does not move the first argument, if you want that to
+      # be on a separate line, see `Layout/FirstMethodParameterLineBreak`.
+      #
+      # @example
+      #
+      #   # bad
+      #   def foo(a, b,
+      #     c
+      #   )
+      #   end
+      #
+      #   # good
+      #   def foo(
```

Comment:
I'd also add some example showing that it's fine to have multiple params if they are all on the same line.

Reply:
Good call, also updated `MultilineMethodParameterLineBreaks` for consistency!